src/seedu/addressbook/commands/FindByTagCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import java.util.*;
+
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * Finds and lists all persons in address book who have at least 1 tag which matches 1 of the specified keywords.
+ * Keyword matching is case sensitive.
+ */
+public class FindByTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "findByTag";
```

Comment:
Should follow convention of other commands- viewbytag or viewtags etc. viewall cmd doesn't have camelcase

Reply:
Thanks @eugenefdw I have changed it accordingly.